src/navigation/BottomTabs/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {PrimaryStackNavigator} from '../PrimaryStack';
+import {BottomTabRoute} from './BottomTabNavigator.types';
+import {CustomIcon, CustomIconNames} from '../../components';
+import ProfileScreen from '../../screens/ProfileScreen/ProfileScreen';
+
+const BottomTabs = createBottomTabNavigator();
+
+const BottomTabNavigator = () => {
+  const tabBarIcon = (props: {
```

Comment:
Can we create a type called TabBarIconProps here

Reply:
Added.